2020-06-13T15:25:21.5563834-04:00  [INF] Tarea ejecutada: 13/06/2020 15:25:21 (af507e53)
2020-06-13T15:25:31.1570257-04:00  [WRN] No type was specified for the decimal column '"Signo"' on entity type '"AsientosView"'. This will cause values to be silently truncated if they do not fit in the default precision and scale. Explicitly specify the SQL server column type that can accommodate all the values using 'HasColumnType()'. (3a0db05f)
2020-06-13T15:25:31.1571878-04:00  [WRN] No type was specified for the decimal column '"n_importe"' on entity type '"AsientosView"'. This will cause values to be silently truncated if they do not fit in the default precision and scale. Explicitly specify the SQL server column type that can accommodate all the values using 'HasColumnType()'. (3a0db05f)
2020-06-13T15:25:31.1572188-04:00  [WRN] No type was specified for the decimal column '"n_importe_anx"' on entity type '"AsientosView"'. This will cause values to be silently truncated if they do not fit in the default precision and scale. Explicitly specify the SQL server column type that can accommodate all the values using 'HasColumnType()'. (3a0db05f)
2020-06-13T15:25:31.1572377-04:00  [WRN] No type was specified for the decimal column '"n_importemo"' on entity type '"AsientosView"'. This will cause values to be silently truncated if they do not fit in the default precision and scale. Explicitly specify the SQL server column type that can accommodate all the values using 'HasColumnType()'. (3a0db05f)
2020-06-13T15:25:40.3653618-04:00  [INF] Tarea ejecutada: 13/06/2020 15:25:40 (12adfc78)
2020-06-13T15:25:40.7432048-04:00  [INF] Tarea ejecutada: 13/06/2020 15:25:40 (12adfc78)
2020-06-13T15:25:41.1782219-04:00  [WRN] The LINQ expression '"GroupBy([c.IdcomprobanteNavigation], _Include(queryContext, [c], new [] {[c.IdcomprobanteNavigation], [c.IdcomprobanteNavigation.ConComprobanteoperacion], [c.IdcomprobanteNavigation.ConComprobanteoperacion.IdusuarioNavigation], [c.IdcomprobanteNavigation.ConComprobanteoperacion.IdperiodoNavigation], [c.IdcuentaNavigation], [c.IdcuentaNavigation.IdaperturaNavigation], [c.IdcuentaNavigation.IdaperturaNavigation.IdmascaraNavigation]}, (queryContext, entity, included) => { ... }))"' could not be translated and will be evaluated locally. (011792c6)
2020-06-13T15:25:44.9340879-04:00  [FTL] An error occurred starting the application (dd5d7f26)
System.AggregateException: One or more errors occurred. (One or more errors occurred. (Se ha intentado una operación de socket en un host no accesible)) ---> System.AggregateException: One or more errors occurred. (Se ha intentado una operación de socket en un host no accesible) ---> System.Net.Http.HttpRequestException: Se ha intentado una operación de socket en un host no accesible ---> System.Net.Sockets.SocketException: Se ha intentado una operación de socket en un host no accesible
   at System.Net.Http.ConnectHelper.ConnectAsync(String host, Int32 port, CancellationToken cancellationToken)
   --- End of inner exception stack trace ---
   at System.Net.Http.ConnectHelper.ConnectAsync(String host, Int32 port, CancellationToken cancellationToken)
   at System.Threading.Tasks.ValueTask`1.get_Result()
   at System.Net.Http.HttpConnectionPool.CreateConnectionAsync(HttpRequestMessage request, CancellationToken cancellationToken)
   at System.Threading.Tasks.ValueTask`1.get_Result()
   at System.Net.Http.HttpConnectionPool.WaitForCreatedConnectionAsync(ValueTask`1 creationTask)
   at System.Threading.Tasks.ValueTask`1.get_Result()
   at System.Net.Http.HttpConnectionPool.SendWithRetryAsync(HttpRequestMessage request, Boolean doRequestAuth, CancellationToken cancellationToken)
   at System.Net.Http.HttpConnectionPool.EstablishProxyTunnel(CancellationToken cancellationToken)
   at System.Threading.Tasks.ValueTask`1.get_Result()
   at System.Net.Http.HttpConnectionPool.CreateConnectionAsync(HttpRequestMessage request, CancellationToken cancellationToken)
   at System.Threading.Tasks.ValueTask`1.get_Result()
   at System.Net.Http.HttpConnectionPool.WaitForCreatedConnectionAsync(ValueTask`1 creationTask)
   at System.Threading.Tasks.ValueTask`1.get_Result()
   at System.Net.Http.HttpConnectionPool.SendWithRetryAsync(HttpRequestMessage request, Boolean doRequestAuth, CancellationToken cancellationToken)
   at System.Net.Http.RedirectHandler.SendAsync(HttpRequestMessage request, CancellationToken cancellationToken)
   at System.Net.Http.HttpClient.FinishSendAsyncBuffered(Task`1 sendTask, HttpRequestMessage request, CancellationTokenSource cts, Boolean disposeCts)
   --- End of inner exception stack trace ---
   at System.Threading.Tasks.Task`1.GetResultCore(Boolean waitCompletionNotification)
   at ImportadorDatos.Jobs.ImportarVersat.ImportarCuentasAsync() in D:\Alejandro\New Folder\Git\opplat\src\ImportadorDatos\Jobs\ImportarVersat.cs:line 66
   at ImportadorDatos.HostedServices.UpdateCuentasHostedService.ExecuteAsync(CancellationToken stoppingToken) in D:\Alejandro\New Folder\Git\opplat\src\ImportadorDatos\HostedServices\UpdateCuentasHostedService.cs:line 33
   at Microsoft.AspNetCore.Hosting.Internal.HostedServiceExecutor.ExecuteAsync(Func`2 callback)
   --- End of inner exception stack trace ---
   at Microsoft.AspNetCore.Hosting.Internal.HostedServiceExecutor.ExecuteAsync(Func`2 callback)
   at Microsoft.AspNetCore.Hosting.Internal.HostedServiceExecutor.StartAsync(CancellationToken token)
---> (Inner Exception #0) System.AggregateException: One or more errors occurred. (Se ha intentado una operación de socket en un host no accesible) ---> System.Net.Http.HttpRequestException: Se ha intentado una operación de socket en un host no accesible ---> System.Net.Sockets.SocketException: Se ha intentado una operación de socket en un host no accesible
   at System.Net.Http.ConnectHelper.ConnectAsync(String host, Int32 port, CancellationToken cancellationToken)
   --- End of inner exception stack trace ---
   at System.Net.Http.ConnectHelper.ConnectAsync(String host, Int32 port, CancellationToken cancellationToken)
   at System.Threading.Tasks.ValueTask`1.get_Result()
   at System.Net.Http.HttpConnectionPool.CreateConnectionAsync(HttpRequestMessage request, CancellationToken cancellationToken)
   at System.Threading.Tasks.ValueTask`1.get_Result()
   at System.Net.Http.HttpConnectionPool.WaitForCreatedConnectionAsync(ValueTask`1 creationTask)
   at System.Threading.Tasks.ValueTask`1.get_Result()
   at System.Net.Http.HttpConnectionPool.SendWithRetryAsync(HttpRequestMessage request, Boolean doRequestAuth, CancellationToken cancellationToken)
   at System.Net.Http.HttpConnectionPool.EstablishProxyTunnel(CancellationToken cancellationToken)
   at System.Threading.Tasks.ValueTask`1.get_Result()
   at System.Net.Http.HttpConnectionPool.CreateConnectionAsync(HttpRequestMessage request, CancellationToken cancellationToken)
   at System.Threading.Tasks.ValueTask`1.get_Result()
   at System.Net.Http.HttpConnectionPool.WaitForCreatedConnectionAsync(ValueTask`1 creationTask)
   at System.Threading.Tasks.ValueTask`1.get_Result()
   at System.Net.Http.HttpConnectionPool.SendWithRetryAsync(HttpRequestMessage request, Boolean doRequestAuth, CancellationToken cancellationToken)
   at System.Net.Http.RedirectHandler.SendAsync(HttpRequestMessage request, CancellationToken cancellationToken)
   at System.Net.Http.HttpClient.FinishSendAsyncBuffered(Task`1 sendTask, HttpRequestMessage request, CancellationTokenSource cts, Boolean disposeCts)
   --- End of inner exception stack trace ---
   at System.Threading.Tasks.Task`1.GetResultCore(Boolean waitCompletionNotification)
   at ImportadorDatos.Jobs.ImportarVersat.ImportarCuentasAsync() in D:\Alejandro\New Folder\Git\opplat\src\ImportadorDatos\Jobs\ImportarVersat.cs:line 66
   at ImportadorDatos.HostedServices.UpdateCuentasHostedService.ExecuteAsync(CancellationToken stoppingToken) in D:\Alejandro\New Folder\Git\opplat\src\ImportadorDatos\HostedServices\UpdateCuentasHostedService.cs:line 33
   at Microsoft.AspNetCore.Hosting.Internal.HostedServiceExecutor.ExecuteAsync(Func`2 callback)
---> (Inner Exception #0) System.Net.Http.HttpRequestException: Se ha intentado una operación de socket en un host no accesible ---> System.Net.Sockets.SocketException: Se ha intentado una operación de socket en un host no accesible
   at System.Net.Http.ConnectHelper.ConnectAsync(String host, Int32 port, CancellationToken cancellationToken)
   --- End of inner exception stack trace ---
   at System.Net.Http.ConnectHelper.ConnectAsync(String host, Int32 port, CancellationToken cancellationToken)
   at System.Threading.Tasks.ValueTask`1.get_Result()
   at System.Net.Http.HttpConnectionPool.CreateConnectionAsync(HttpRequestMessage request, CancellationToken cancellationToken)
   at System.Threading.Tasks.ValueTask`1.get_Result()
   at System.Net.Http.HttpConnectionPool.WaitForCreatedConnectionAsync(ValueTask`1 creationTask)
   at System.Threading.Tasks.ValueTask`1.get_Result()
   at System.Net.Http.HttpConnectionPool.SendWithRetryAsync(HttpRequestMessage request, Boolean doRequestAuth, CancellationToken cancellationToken)
   at System.Net.Http.HttpConnectionPool.EstablishProxyTunnel(CancellationToken cancellationToken)
   at System.Threading.Tasks.ValueTask`1.get_Result()
   at System.Net.Http.HttpConnectionPool.CreateConnectionAsync(HttpRequestMessage request, CancellationToken cancellationToken)
   at System.Threading.Tasks.ValueTask`1.get_Result()
   at System.Net.Http.HttpConnectionPool.WaitForCreatedConnectionAsync(ValueTask`1 creationTask)
   at System.Threading.Tasks.ValueTask`1.get_Result()
   at System.Net.Http.HttpConnectionPool.SendWithRetryAsync(HttpRequestMessage request, Boolean doRequestAuth, CancellationToken cancellationToken)
   at System.Net.Http.RedirectHandler.SendAsync(HttpRequestMessage request, CancellationToken cancellationToken)
   at System.Net.Http.HttpClient.FinishSendAsyncBuffered(Task`1 sendTask, HttpRequestMessage request, CancellationTokenSource cts, Boolean disposeCts)<---
<---

2020-06-13T15:26:30.9348492-04:00 0HM0FPUJS49VT:00000003 [ERR] An unhandled exception has occurred while executing the request. (48a46595)
System.AggregateException: One or more errors occurred. (Se ha intentado una operación de socket en un host no accesible) ---> System.Net.Http.HttpRequestException: Se ha intentado una operación de socket en un host no accesible ---> System.Net.Sockets.SocketException: Se ha intentado una operación de socket en un host no accesible
   at System.Net.Http.ConnectHelper.ConnectAsync(String host, Int32 port, CancellationToken cancellationToken)
   --- End of inner exception stack trace ---
   at System.Net.Http.ConnectHelper.ConnectAsync(String host, Int32 port, CancellationToken cancellationToken)
   at System.Threading.Tasks.ValueTask`1.get_Result()
   at System.Net.Http.HttpConnectionPool.CreateConnectionAsync(HttpRequestMessage request, CancellationToken cancellationToken)
   at System.Threading.Tasks.ValueTask`1.get_Result()
   at System.Net.Http.HttpConnectionPool.WaitForCreatedConnectionAsync(ValueTask`1 creationTask)
   at System.Threading.Tasks.ValueTask`1.get_Result()
   at System.Net.Http.HttpConnectionPool.SendWithRetryAsync(HttpRequestMessage request, Boolean doRequestAuth, CancellationToken cancellationToken)
   at System.Net.Http.HttpConnectionPool.EstablishProxyTunnel(CancellationToken cancellationToken)
   at System.Threading.Tasks.ValueTask`1.get_Result()
   at System.Net.Http.HttpConnectionPool.CreateConnectionAsync(HttpRequestMessage request, CancellationToken cancellationToken)
   at System.Threading.Tasks.ValueTask`1.get_Result()
   at System.Net.Http.HttpConnectionPool.WaitForCreatedConnectionAsync(ValueTask`1 creationTask)
   at System.Threading.Tasks.ValueTask`1.get_Result()
   at System.Net.Http.HttpConnectionPool.SendWithRetryAsync(HttpRequestMessage request, Boolean doRequestAuth, CancellationToken cancellationToken)
   at System.Net.Http.RedirectHandler.SendAsync(HttpRequestMessage request, CancellationToken cancellationToken)
   at System.Net.Http.HttpClient.FinishSendAsyncBuffered(Task`1 sendTask, HttpRequestMessage request, CancellationTokenSource cts, Boolean disposeCts)
   --- End of inner exception stack trace ---
   at System.Threading.Tasks.Task`1.GetResultCore(Boolean waitCompletionNotification)
   at FinanzasWebApi.Helper.GetPlanEgresosPeriodo.Get(Int32 year, IConfiguration config) in D:\Alejandro\New Folder\Git\opplat\src\FinanzasWebApi\Helper\GetPlanEgresosPeriodo.cs:line 21
   at FinanzasWebApi.Helper.ObtenerPlanGI.ObtenerTotalPlanEgresos(Int32 year, Int32 meses) in D:\Alejandro\New Folder\Git\opplat\src\FinanzasWebApi\Helper\ObtenerPlanGI.cs:line 257
   at FinanzasWebApi.Controllers.ReporteIngresosGastosController.EgresosTotal(String años) in D:\Alejandro\New Folder\Git\opplat\src\FinanzasWebApi\Controllers\ReporteIngresosGastosController.cs:line 109
   at lambda_method(Closure , Object , Object[] )
   at Microsoft.Extensions.Internal.ObjectMethodExecutor.Execute(Object target, Object[] parameters)
   at Microsoft.AspNetCore.Mvc.Internal.ActionMethodExecutor.SyncObjectResultExecutor.Execute(IActionResultTypeMapper mapper, ObjectMethodExecutor executor, Object controller, Object[] arguments)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeActionMethodAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeNextActionFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Rethrow(ActionExecutedContext context)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeInnerFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeNextResourceFilter()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.Rethrow(ResourceExecutedContext context)
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeFilterPipelineAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeAsync()
   at Microsoft.AspNetCore.Routing.EndpointMiddleware.Invoke(HttpContext httpContext)
   at Microsoft.AspNetCore.Routing.EndpointRoutingMiddleware.Invoke(HttpContext httpContext)
   at Microsoft.AspNetCore.Cors.Infrastructure.CorsMiddleware.InvokeCore(HttpContext context)
   at Microsoft.AspNetCore.Builder.RouterMiddleware.Invoke(HttpContext httpContext)
   at Microsoft.AspNetCore.StaticFiles.StaticFileMiddleware.Invoke(HttpContext context)
   at Swashbuckle.AspNetCore.SwaggerUI.SwaggerUIMiddleware.Invoke(HttpContext httpContext)
   at Swashbuckle.AspNetCore.Swagger.SwaggerMiddleware.Invoke(HttpContext httpContext, ISwaggerProvider swaggerProvider)
   at Microsoft.AspNetCore.StaticFiles.StaticFileMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware.Invoke(HttpContext context)
---> (Inner Exception #0) System.Net.Http.HttpRequestException: Se ha intentado una operación de socket en un host no accesible ---> System.Net.Sockets.SocketException: Se ha intentado una operación de socket en un host no accesible
   at System.Net.Http.ConnectHelper.ConnectAsync(String host, Int32 port, CancellationToken cancellationToken)
   --- End of inner exception stack trace ---
   at System.Net.Http.ConnectHelper.ConnectAsync(String host, Int32 port, CancellationToken cancellationToken)
   at System.Threading.Tasks.ValueTask`1.get_Result()
   at System.Net.Http.HttpConnectionPool.CreateConnectionAsync(HttpRequestMessage request, CancellationToken cancellationToken)
   at System.Threading.Tasks.ValueTask`1.get_Result()
   at System.Net.Http.HttpConnectionPool.WaitForCreatedConnectionAsync(ValueTask`1 creationTask)
   at System.Threading.Tasks.ValueTask`1.get_Result()
   at System.Net.Http.HttpConnectionPool.SendWithRetryAsync(HttpRequestMessage request, Boolean doRequestAuth, CancellationToken cancellationToken)
   at System.Net.Http.HttpConnectionPool.EstablishProxyTunnel(CancellationToken cancellationToken)
   at System.Threading.Tasks.ValueTask`1.get_Result()
   at System.Net.Http.HttpConnectionPool.CreateConnectionAsync(HttpRequestMessage request, CancellationToken cancellationToken)
   at System.Threading.Tasks.ValueTask`1.get_Result()
   at System.Net.Http.HttpConnectionPool.WaitForCreatedConnectionAsync(ValueTask`1 creationTask)
   at System.Threading.Tasks.ValueTask`1.get_Result()
   at System.Net.Http.HttpConnectionPool.SendWithRetryAsync(HttpRequestMessage request, Boolean doRequestAuth, CancellationToken cancellationToken)
   at System.Net.Http.RedirectHandler.SendAsync(HttpRequestMessage request, CancellationToken cancellationToken)
   at System.Net.Http.HttpClient.FinishSendAsyncBuffered(Task`1 sendTask, HttpRequestMessage request, CancellationTokenSource cts, Boolean disposeCts)<---

2020-06-13T15:26:30.9348492-04:00 0HM0FPUJS49VV:00000002 [ERR] An unhandled exception has occurred while executing the request. (48a46595)
System.AggregateException: One or more errors occurred. (Se ha intentado una operación de socket en un host no accesible) ---> System.Net.Http.HttpRequestException: Se ha intentado una operación de socket en un host no accesible ---> System.Net.Sockets.SocketException: Se ha intentado una operación de socket en un host no accesible
   at System.Net.Http.ConnectHelper.ConnectAsync(String host, Int32 port, CancellationToken cancellationToken)
   --- End of inner exception stack trace ---
   at System.Net.Http.ConnectHelper.ConnectAsync(String host, Int32 port, CancellationToken cancellationToken)
   at System.Threading.Tasks.ValueTask`1.get_Result()
   at System.Net.Http.HttpConnectionPool.CreateConnectionAsync(HttpRequestMessage request, CancellationToken cancellationToken)
   at System.Threading.Tasks.ValueTask`1.get_Result()
   at System.Net.Http.HttpConnectionPool.WaitForCreatedConnectionAsync(ValueTask`1 creationTask)
   at System.Threading.Tasks.ValueTask`1.get_Result()
   at System.Net.Http.HttpConnectionPool.SendWithRetryAsync(HttpRequestMessage request, Boolean doRequestAuth, CancellationToken cancellationToken)
   at System.Net.Http.HttpConnectionPool.EstablishProxyTunnel(CancellationToken cancellationToken)
   at System.Threading.Tasks.ValueTask`1.get_Result()
   at System.Net.Http.HttpConnectionPool.CreateConnectionAsync(HttpRequestMessage request, CancellationToken cancellationToken)
   at System.Threading.Tasks.ValueTask`1.get_Result()
   at System.Net.Http.HttpConnectionPool.WaitForCreatedConnectionAsync(ValueTask`1 creationTask)
   at System.Threading.Tasks.ValueTask`1.get_Result()
   at System.Net.Http.HttpConnectionPool.SendWithRetryAsync(HttpRequestMessage request, Boolean doRequestAuth, CancellationToken cancellationToken)
   at System.Net.Http.RedirectHandler.SendAsync(HttpRequestMessage request, CancellationToken cancellationToken)
   at System.Net.Http.HttpClient.FinishSendAsyncBuffered(Task`1 sendTask, HttpRequestMessage request, CancellationTokenSource cts, Boolean disposeCts)
   --- End of inner exception stack trace ---
   at System.Threading.Tasks.Task`1.GetResultCore(Boolean waitCompletionNotification)
   at FinanzasWebApi.Helper.GetPlanIngresosPeriodo.Get(Int32 year, IConfiguration config) in D:\Alejandro\New Folder\Git\opplat\src\FinanzasWebApi\Helper\GetPlanIngresosPeriodo.cs:line 21
   at FinanzasWebApi.Helper.ObtenerPlanGI.ObtenerTotalPlanIngresos(Int32 year, Int32 meses) in D:\Alejandro\New Folder\Git\opplat\src\FinanzasWebApi\Helper\ObtenerPlanGI.cs:line 122
   at FinanzasWebApi.Helper.ObtenerPlanGI.ObtenerTotalPlanUtilidades(Int32 year, Int32 meses) in D:\Alejandro\New Folder\Git\opplat\src\FinanzasWebApi\Helper\ObtenerPlanGI.cs:line 471
   at FinanzasWebApi.Controllers.ReporteIngresosGastosController.UtilidadesTotal(String años) in D:\Alejandro\New Folder\Git\opplat\src\FinanzasWebApi\Controllers\ReporteIngresosGastosController.cs:line 153
   at lambda_method(Closure , Object , Object[] )
   at Microsoft.Extensions.Internal.ObjectMethodExecutor.Execute(Object target, Object[] parameters)
   at Microsoft.AspNetCore.Mvc.Internal.ActionMethodExecutor.SyncObjectResultExecutor.Execute(IActionResultTypeMapper mapper, ObjectMethodExecutor executor, Object controller, Object[] arguments)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeActionMethodAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeNextActionFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Rethrow(ActionExecutedContext context)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeInnerFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeNextResourceFilter()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.Rethrow(ResourceExecutedContext context)
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeFilterPipelineAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeAsync()
   at Microsoft.AspNetCore.Routing.EndpointMiddleware.Invoke(HttpContext httpContext)
   at Microsoft.AspNetCore.Routing.EndpointRoutingMiddleware.Invoke(HttpContext httpContext)
   at Microsoft.AspNetCore.Cors.Infrastructure.CorsMiddleware.InvokeCore(HttpContext context)
   at Microsoft.AspNetCore.Builder.RouterMiddleware.Invoke(HttpContext httpContext)
   at Microsoft.AspNetCore.StaticFiles.StaticFileMiddleware.Invoke(HttpContext context)
   at Swashbuckle.AspNetCore.SwaggerUI.SwaggerUIMiddleware.Invoke(HttpContext httpContext)
   at Swashbuckle.AspNetCore.Swagger.SwaggerMiddleware.Invoke(HttpContext httpContext, ISwaggerProvider swaggerProvider)
   at Microsoft.AspNetCore.StaticFiles.StaticFileMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware.Invoke(HttpContext context)
---> (Inner Exception #0) System.Net.Http.HttpRequestException: Se ha intentado una operación de socket en un host no accesible ---> System.Net.Sockets.SocketException: Se ha intentado una operación de socket en un host no accesible
   at System.Net.Http.ConnectHelper.ConnectAsync(String host, Int32 port, CancellationToken cancellationToken)
   --- End of inner exception stack trace ---
   at System.Net.Http.ConnectHelper.ConnectAsync(String host, Int32 port, CancellationToken cancellationToken)
   at System.Threading.Tasks.ValueTask`1.get_Result()
   at System.Net.Http.HttpConnectionPool.CreateConnectionAsync(HttpRequestMessage request, CancellationToken cancellationToken)
   at System.Threading.Tasks.ValueTask`1.get_Result()
   at System.Net.Http.HttpConnectionPool.WaitForCreatedConnectionAsync(ValueTask`1 creationTask)
   at System.Threading.Tasks.ValueTask`1.get_Result()
   at System.Net.Http.HttpConnectionPool.SendWithRetryAsync(HttpRequestMessage request, Boolean doRequestAuth, CancellationToken cancellationToken)
   at System.Net.Http.HttpConnectionPool.EstablishProxyTunnel(CancellationToken cancellationToken)
   at System.Threading.Tasks.ValueTask`1.get_Result()
   at System.Net.Http.HttpConnectionPool.CreateConnectionAsync(HttpRequestMessage request, CancellationToken cancellationToken)
   at System.Threading.Tasks.ValueTask`1.get_Result()
   at System.Net.Http.HttpConnectionPool.WaitForCreatedConnectionAsync(ValueTask`1 creationTask)
   at System.Threading.Tasks.ValueTask`1.get_Result()
   at System.Net.Http.HttpConnectionPool.SendWithRetryAsync(HttpRequestMessage request, Boolean doRequestAuth, CancellationToken cancellationToken)
   at System.Net.Http.RedirectHandler.SendAsync(HttpRequestMessage request, CancellationToken cancellationToken)
   at System.Net.Http.HttpClient.FinishSendAsyncBuffered(Task`1 sendTask, HttpRequestMessage request, CancellationTokenSource cts, Boolean disposeCts)<---

2020-06-13T15:26:30.9352583-04:00 0HM0FPUJS49VS:00000004 [ERR] An unhandled exception has occurred while executing the request. (48a46595)
System.AggregateException: One or more errors occurred. (Se ha intentado una operación de socket en un host no accesible) ---> System.Net.Http.HttpRequestException: Se ha intentado una operación de socket en un host no accesible ---> System.Net.Sockets.SocketException: Se ha intentado una operación de socket en un host no accesible
   at System.Net.Http.ConnectHelper.ConnectAsync(String host, Int32 port, CancellationToken cancellationToken)
   --- End of inner exception stack trace ---
   at System.Net.Http.ConnectHelper.ConnectAsync(String host, Int32 port, CancellationToken cancellationToken)
   at System.Threading.Tasks.ValueTask`1.get_Result()
   at System.Net.Http.HttpConnectionPool.CreateConnectionAsync(HttpRequestMessage request, CancellationToken cancellationToken)
   at System.Threading.Tasks.ValueTask`1.get_Result()
   at System.Net.Http.HttpConnectionPool.WaitForCreatedConnectionAsync(ValueTask`1 creationTask)
   at System.Threading.Tasks.ValueTask`1.get_Result()
   at System.Net.Http.HttpConnectionPool.SendWithRetryAsync(HttpRequestMessage request, Boolean doRequestAuth, CancellationToken cancellationToken)
   at System.Net.Http.HttpConnectionPool.EstablishProxyTunnel(CancellationToken cancellationToken)
   at System.Threading.Tasks.ValueTask`1.get_Result()
   at System.Net.Http.HttpConnectionPool.CreateConnectionAsync(HttpRequestMessage request, CancellationToken cancellationToken)
   at System.Threading.Tasks.ValueTask`1.get_Result()
   at System.Net.Http.HttpConnectionPool.WaitForCreatedConnectionAsync(ValueTask`1 creationTask)
   at System.Threading.Tasks.ValueTask`1.get_Result()
   at System.Net.Http.HttpConnectionPool.SendWithRetryAsync(HttpRequestMessage request, Boolean doRequestAuth, CancellationToken cancellationToken)
   at System.Net.Http.RedirectHandler.SendAsync(HttpRequestMessage request, CancellationToken cancellationToken)
   at System.Net.Http.HttpClient.FinishSendAsyncBuffered(Task`1 sendTask, HttpRequestMessage request, CancellationTokenSource cts, Boolean disposeCts)
   --- End of inner exception stack trace ---
   at System.Threading.Tasks.Task`1.GetResultCore(Boolean waitCompletionNotification)
   at FinanzasWebApi.Helper.GetPlanIngresosPeriodo.Get(Int32 year, IConfiguration config) in D:\Alejandro\New Folder\Git\opplat\src\FinanzasWebApi\Helper\GetPlanIngresosPeriodo.cs:line 21
   at FinanzasWebApi.Helper.ObtenerPlanGI.ObtenerTotalPlanIngresos(Int32 year, Int32 meses) in D:\Alejandro\New Folder\Git\opplat\src\FinanzasWebApi\Helper\ObtenerPlanGI.cs:line 122
   at FinanzasWebApi.Controllers.ReporteIngresosGastosController.IngresosTotal(String años) in D:\Alejandro\New Folder\Git\opplat\src\FinanzasWebApi\Controllers\ReporteIngresosGastosController.cs:line 64
   at lambda_method(Closure , Object , Object[] )
   at Microsoft.Extensions.Internal.ObjectMethodExecutor.Execute(Object target, Object[] parameters)
   at Microsoft.AspNetCore.Mvc.Internal.ActionMethodExecutor.SyncActionResultExecutor.Execute(IActionResultTypeMapper mapper, ObjectMethodExecutor executor, Object controller, Object[] arguments)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeActionMethodAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeNextActionFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Rethrow(ActionExecutedContext context)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeInnerFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeNextResourceFilter()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.Rethrow(ResourceExecutedContext context)
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeFilterPipelineAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeAsync()
   at Microsoft.AspNetCore.Routing.EndpointMiddleware.Invoke(HttpContext httpContext)
   at Microsoft.AspNetCore.Routing.EndpointRoutingMiddleware.Invoke(HttpContext httpContext)
   at Microsoft.AspNetCore.Cors.Infrastructure.CorsMiddleware.InvokeCore(HttpContext context)
   at Microsoft.AspNetCore.Builder.RouterMiddleware.Invoke(HttpContext httpContext)
   at Microsoft.AspNetCore.StaticFiles.StaticFileMiddleware.Invoke(HttpContext context)
   at Swashbuckle.AspNetCore.SwaggerUI.SwaggerUIMiddleware.Invoke(HttpContext httpContext)
   at Swashbuckle.AspNetCore.Swagger.SwaggerMiddleware.Invoke(HttpContext httpContext, ISwaggerProvider swaggerProvider)
   at Microsoft.AspNetCore.StaticFiles.StaticFileMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware.Invoke(HttpContext context)
---> (Inner Exception #0) System.Net.Http.HttpRequestException: Se ha intentado una operación de socket en un host no accesible ---> System.Net.Sockets.SocketException: Se ha intentado una operación de socket en un host no accesible
   at System.Net.Http.ConnectHelper.ConnectAsync(String host, Int32 port, CancellationToken cancellationToken)
   --- End of inner exception stack trace ---
   at System.Net.Http.ConnectHelper.ConnectAsync(String host, Int32 port, CancellationToken cancellationToken)
   at System.Threading.Tasks.ValueTask`1.get_Result()
   at System.Net.Http.HttpConnectionPool.CreateConnectionAsync(HttpRequestMessage request, CancellationToken cancellationToken)
   at System.Threading.Tasks.ValueTask`1.get_Result()
   at System.Net.Http.HttpConnectionPool.WaitForCreatedConnectionAsync(ValueTask`1 creationTask)
   at System.Threading.Tasks.ValueTask`1.get_Result()
   at System.Net.Http.HttpConnectionPool.SendWithRetryAsync(HttpRequestMessage request, Boolean doRequestAuth, CancellationToken cancellationToken)
   at System.Net.Http.HttpConnectionPool.EstablishProxyTunnel(CancellationToken cancellationToken)
   at System.Threading.Tasks.ValueTask`1.get_Result()
   at System.Net.Http.HttpConnectionPool.CreateConnectionAsync(HttpRequestMessage request, CancellationToken cancellationToken)
   at System.Threading.Tasks.ValueTask`1.get_Result()
   at System.Net.Http.HttpConnectionPool.WaitForCreatedConnectionAsync(ValueTask`1 creationTask)
   at System.Threading.Tasks.ValueTask`1.get_Result()
   at System.Net.Http.HttpConnectionPool.SendWithRetryAsync(HttpRequestMessage request, Boolean doRequestAuth, CancellationToken cancellationToken)
   at System.Net.Http.RedirectHandler.SendAsync(HttpRequestMessage request, CancellationToken cancellationToken)
   at System.Net.Http.HttpClient.FinishSendAsyncBuffered(Task`1 sendTask, HttpRequestMessage request, CancellationTokenSource cts, Boolean disposeCts)<---

2020-06-13T15:26:31.0130601-04:00 0HM0FPUJS4A00:00000001 [ERR] An unhandled exception has occurred while executing the request. (48a46595)
System.AggregateException: One or more errors occurred. (Se ha intentado una operación de socket en un host no accesible) ---> System.Net.Http.HttpRequestException: Se ha intentado una operación de socket en un host no accesible ---> System.Net.Sockets.SocketException: Se ha intentado una operación de socket en un host no accesible
   at System.Net.Http.ConnectHelper.ConnectAsync(String host, Int32 port, CancellationToken cancellationToken)
   --- End of inner exception stack trace ---
   at System.Net.Http.ConnectHelper.ConnectAsync(String host, Int32 port, CancellationToken cancellationToken)
   at System.Threading.Tasks.ValueTask`1.get_Result()
   at System.Net.Http.HttpConnectionPool.CreateConnectionAsync(HttpRequestMessage request, CancellationToken cancellationToken)
   at System.Threading.Tasks.ValueTask`1.get_Result()
   at System.Net.Http.HttpConnectionPool.WaitForCreatedConnectionAsync(ValueTask`1 creationTask)
   at System.Threading.Tasks.ValueTask`1.get_Result()
   at System.Net.Http.HttpConnectionPool.SendWithRetryAsync(HttpRequestMessage request, Boolean doRequestAuth, CancellationToken cancellationToken)
   at System.Net.Http.HttpConnectionPool.EstablishProxyTunnel(CancellationToken cancellationToken)
   at System.Threading.Tasks.ValueTask`1.get_Result()
   at System.Net.Http.HttpConnectionPool.CreateConnectionAsync(HttpRequestMessage request, CancellationToken cancellationToken)
   at System.Threading.Tasks.ValueTask`1.get_Result()
   at System.Net.Http.HttpConnectionPool.WaitForCreatedConnectionAsync(ValueTask`1 creationTask)
   at System.Threading.Tasks.ValueTask`1.get_Result()
   at System.Net.Http.HttpConnectionPool.SendWithRetryAsync(HttpRequestMessage request, Boolean doRequestAuth, CancellationToken cancellationToken)
   at System.Net.Http.RedirectHandler.SendAsync(HttpRequestMessage request, CancellationToken cancellationToken)
   at System.Net.Http.HttpClient.FinishSendAsyncBuffered(Task`1 sendTask, HttpRequestMessage request, CancellationTokenSource cts, Boolean disposeCts)
   --- End of inner exception stack trace ---
   at System.Threading.Tasks.Task`1.GetResultCore(Boolean waitCompletionNotification)
   at FinanzasWebApi.Helper.GetPlanIngresosPeriodo.Get(Int32 year, IConfiguration config) in D:\Alejandro\New Folder\Git\opplat\src\FinanzasWebApi\Helper\GetPlanIngresosPeriodo.cs:line 21
   at FinanzasWebApi.Helper.ObtenerValuesEnVariablesEstadoFinanciero.ObtenerVentas(Int32 year, Int32 mes) in D:\Alejandro\New Folder\Git\opplat\src\FinanzasWebApi\Helper\ObtenerValuesEnVariablesEstadoFinanciero.cs:line 412
   at FinanzasWebApi.Controllers.RazonesFinancierasController.mesActual(Int32 años, Int32 meses) in D:\Alejandro\New Folder\Git\opplat\src\FinanzasWebApi\Controllers\RazonesFinancierasController.cs:line 212
   at lambda_method(Closure , Object , Object[] )
   at Microsoft.Extensions.Internal.ObjectMethodExecutor.Execute(Object target, Object[] parameters)
   at Microsoft.AspNetCore.Mvc.Internal.ActionMethodExecutor.SyncObjectResultExecutor.Execute(IActionResultTypeMapper mapper, ObjectMethodExecutor executor, Object controller, Object[] arguments)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeActionMethodAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeNextActionFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Rethrow(ActionExecutedContext context)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeInnerFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeNextResourceFilter()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.Rethrow(ResourceExecutedContext context)
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeFilterPipelineAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeAsync()
   at Microsoft.AspNetCore.Routing.EndpointMiddleware.Invoke(HttpContext httpContext)
   at Microsoft.AspNetCore.Routing.EndpointRoutingMiddleware.Invoke(HttpContext httpContext)
   at Microsoft.AspNetCore.Cors.Infrastructure.CorsMiddleware.InvokeCore(HttpContext context)
   at Microsoft.AspNetCore.Builder.RouterMiddleware.Invoke(HttpContext httpContext)
   at Microsoft.AspNetCore.StaticFiles.StaticFileMiddleware.Invoke(HttpContext context)
   at Swashbuckle.AspNetCore.SwaggerUI.SwaggerUIMiddleware.Invoke(HttpContext httpContext)
   at Swashbuckle.AspNetCore.Swagger.SwaggerMiddleware.Invoke(HttpContext httpContext, ISwaggerProvider swaggerProvider)
   at Microsoft.AspNetCore.StaticFiles.StaticFileMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware.Invoke(HttpContext context)
---> (Inner Exception #0) System.Net.Http.HttpRequestException: Se ha intentado una operación de socket en un host no accesible ---> System.Net.Sockets.SocketException: Se ha intentado una operación de socket en un host no accesible
   at System.Net.Http.ConnectHelper.ConnectAsync(String host, Int32 port, CancellationToken cancellationToken)
   --- End of inner exception stack trace ---
   at System.Net.Http.ConnectHelper.ConnectAsync(String host, Int32 port, CancellationToken cancellationToken)
   at System.Threading.Tasks.ValueTask`1.get_Result()
   at System.Net.Http.HttpConnectionPool.CreateConnectionAsync(HttpRequestMessage request, CancellationToken cancellationToken)
   at System.Threading.Tasks.ValueTask`1.get_Result()
   at System.Net.Http.HttpConnectionPool.WaitForCreatedConnectionAsync(ValueTask`1 creationTask)
   at System.Threading.Tasks.ValueTask`1.get_Result()
   at System.Net.Http.HttpConnectionPool.SendWithRetryAsync(HttpRequestMessage request, Boolean doRequestAuth, CancellationToken cancellationToken)
   at System.Net.Http.HttpConnectionPool.EstablishProxyTunnel(CancellationToken cancellationToken)
   at System.Threading.Tasks.ValueTask`1.get_Result()
   at System.Net.Http.HttpConnectionPool.CreateConnectionAsync(HttpRequestMessage request, CancellationToken cancellationToken)
   at System.Threading.Tasks.ValueTask`1.get_Result()
   at System.Net.Http.HttpConnectionPool.WaitForCreatedConnectionAsync(ValueTask`1 creationTask)
   at System.Threading.Tasks.ValueTask`1.get_Result()
   at System.Net.Http.HttpConnectionPool.SendWithRetryAsync(HttpRequestMessage request, Boolean doRequestAuth, CancellationToken cancellationToken)
   at System.Net.Http.RedirectHandler.SendAsync(HttpRequestMessage request, CancellationToken cancellationToken)
   at System.Net.Http.HttpClient.FinishSendAsyncBuffered(Task`1 sendTask, HttpRequestMessage request, CancellationTokenSource cts, Boolean disposeCts)<---

2020-06-13T15:26:33.4562144-04:00 0HM0FPUJS49VV:00000006 [WRN] The LINQ expression '"where (([c].FechaDeVenOferta - DateTime.Now).Days < __8__locals1_tiempoVenOferta_OfertasVencidas_2)"' could not be translated and will be evaluated locally. (011792c6)
2020-06-13T15:26:33.4571768-04:00 0HM0FPUJS49VV:00000006 [WRN] The LINQ expression '"Count()"' could not be translated and will be evaluated locally. (011792c6)
2020-06-13T15:26:33.5042299-04:00 0HM0FPUJS49VV:00000006 [WRN] The LINQ expression '"where (([c].FechaDeVenOferta - DateTime.Now).Days < __tiempoVenOferta_OfertasVencidas_1)"' could not be translated and will be evaluated locally. (011792c6)
2020-06-13T15:26:33.5045516-04:00 0HM0FPUJS49VV:00000006 [WRN] The LINQ expression '"Count()"' could not be translated and will be evaluated locally. (011792c6)
2020-06-13T15:26:33.5095276-04:00 0HM0FPUJS49VV:00000006 [WRN] The LINQ expression '"where (([c].FechaDeVenOferta - DateTime.Now).Days < __tiempoVenOferta_OfertasVencidas_1)"' could not be translated and will be evaluated locally. (011792c6)
2020-06-13T15:26:33.5097713-04:00 0HM0FPUJS49VV:00000006 [WRN] The LINQ expression '"Count()"' could not be translated and will be evaluated locally. (011792c6)
2020-06-13T15:27:07.1810130-04:00 0HM0FPUJS4A00:00000007 [WRN] The Include operation for navigation '"[e].CuentasBancarias"' is unnecessary and was ignored because the navigation is not reachable in the final query results. See https://go.microsoft.com/fwlink/?linkid=850303 for more information. (10efaee6)
2020-06-13T15:27:07.1812524-04:00 0HM0FPUJS4A00:00000007 [WRN] The Include operation for navigation '"[e].Telefonos"' is unnecessary and was ignored because the navigation is not reachable in the final query results. See https://go.microsoft.com/fwlink/?linkid=850303 for more information. (10efaee6)
2020-06-13T15:27:40.0196479-04:00 0HM0FPUJS4A01:00000001 [WRN] The LINQ expression '"where (([c].FechaDeVenOferta - DateTime.Now).Days < __tiempoVenOferta_OfertasVencidas_1)"' could not be translated and will be evaluated locally. (011792c6)
2020-06-13T15:27:40.0202473-04:00 0HM0FPUJS4A01:00000001 [WRN] The LINQ expression '"Count()"' could not be translated and will be evaluated locally. (011792c6)
2020-06-13T15:27:40.0290121-04:00 0HM0FPUJS4A01:00000001 [WRN] The LINQ expression '"where ((([c].FechaDeVenOferta - DateTime.Now).Days >= __tiempoVenOferta_OfertasCasiVencDesde_1) AndAlso (([c].FechaDeVenOferta - DateTime.Now).Days <= __tiempoVenOferta_OfertasCasiVencHasta_2))"' could not be translated and will be evaluated locally. (011792c6)
2020-06-13T15:27:40.0296324-04:00 0HM0FPUJS4A01:00000001 [WRN] The LINQ expression '"Count()"' could not be translated and will be evaluated locally. (011792c6)
2020-06-13T15:27:40.0403436-04:00 0HM0FPUJS4A02:00000001 [WRN] The LINQ expression '"where ((([c].FechaDeVenOferta - DateTime.Now).Days > __tiempoVenOferta_OfertasProxVencDesde_1) AndAlso (([c].FechaDeVenOferta - DateTime.Now).Days <= __tiempoVenOferta_OfertasProxVencHasta_2))"' could not be translated and will be evaluated locally. (011792c6)
2020-06-13T15:27:40.0409249-04:00 0HM0FPUJS4A02:00000001 [WRN] The LINQ expression '"Count()"' could not be translated and will be evaluated locally. (011792c6)
2020-06-13T15:27:40.0502480-04:00 0HM0FPUJS4A02:00000001 [WRN] The LINQ expression '"where (([c].FechaDeVenOferta - DateTime.Now).Days > __tiempoVenOferta_OfertaTiempo_1)"' could not be translated and will be evaluated locally. (011792c6)
2020-06-13T15:27:40.0506778-04:00 0HM0FPUJS4A02:00000001 [WRN] The LINQ expression '"Count()"' could not be translated and will be evaluated locally. (011792c6)
2020-06-13T15:27:40.2757991-04:00 0HM0FPUJS4A04:00000001 [WRN] The LINQ expression '"where ([t].Id == [c].AdminContratoId)"' could not be translated and will be evaluated locally. (011792c6)
2020-06-13T15:27:40.2762117-04:00 0HM0FPUJS4A04:00000001 [WRN] The LINQ expression '"FirstOrDefault()"' could not be translated and will be evaluated locally. (011792c6)
2020-06-13T15:27:40.2764054-04:00 0HM0FPUJS4A04:00000001 [WRN] The LINQ expression '"where ([s].ContratoId == [c].Id)"' could not be translated and will be evaluated locally. (011792c6)
2020-06-13T15:27:40.2766435-04:00 0HM0FPUJS4A04:00000001 [WRN] The LINQ expression '"where ([d].Id == [s].DictaminadorContratoId)"' could not be translated and will be evaluated locally. (011792c6)
2020-06-13T15:27:40.2767667-04:00 0HM0FPUJS4A04:00000001 [WRN] The LINQ expression '"FirstOrDefault()"' could not be translated and will be evaluated locally. (011792c6)
2020-06-13T15:30:44.0573026-04:00  [INF] Tarea ejecutada: 13/06/2020 15:30:44 (d3210ef2)
2020-06-13T15:35:45.3614698-04:00  [INF] Tarea ejecutada: 13/06/2020 15:35:45 (a94eb61e)
2020-06-13T15:37:03.5824282-04:00 0HM0FPUJS4A07:0000000F [WRN] The LINQ expression '"where (([c].FechaDeVenOferta - DateTime.Now).Days < __tiempoVenOferta_OfertasVencidas_1)"' could not be translated and will be evaluated locally. (011792c6)
2020-06-13T15:37:03.5829880-04:00 0HM0FPUJS4A07:0000000F [WRN] The LINQ expression '"Count()"' could not be translated and will be evaluated locally. (011792c6)
2020-06-13T15:37:03.5923072-04:00 0HM0FPUJS4A07:0000000F [WRN] The LINQ expression '"where ((([c].FechaDeVenOferta - DateTime.Now).Days >= __tiempoVenOferta_OfertasCasiVencDesde_1) AndAlso (([c].FechaDeVenOferta - DateTime.Now).Days <= __tiempoVenOferta_OfertasCasiVencHasta_2))"' could not be translated and will be evaluated locally. (011792c6)
2020-06-13T15:37:03.5927695-04:00 0HM0FPUJS4A07:0000000F [WRN] The LINQ expression '"Count()"' could not be translated and will be evaluated locally. (011792c6)
2020-06-13T15:37:03.5930301-04:00 0HM0FPUJS4A06:00000021 [WRN] The LINQ expression '"where ([t].Id == [c].AdminContratoId)"' could not be translated and will be evaluated locally. (011792c6)
2020-06-13T15:37:03.5933170-04:00 0HM0FPUJS4A06:00000021 [WRN] The LINQ expression '"FirstOrDefault()"' could not be translated and will be evaluated locally. (011792c6)
2020-06-13T15:37:03.5935627-04:00 0HM0FPUJS4A06:00000021 [WRN] The LINQ expression '"where ([s].ContratoId == [c].Id)"' could not be translated and will be evaluated locally. (011792c6)
2020-06-13T15:37:03.5939571-04:00 0HM0FPUJS4A06:00000021 [WRN] The LINQ expression '"where ([d].Id == [s].DictaminadorContratoId)"' could not be translated and will be evaluated locally. (011792c6)
2020-06-13T15:37:03.5942075-04:00 0HM0FPUJS4A06:00000021 [WRN] The LINQ expression '"FirstOrDefault()"' could not be translated and will be evaluated locally. (011792c6)
2020-06-13T15:37:03.6014610-04:00 0HM0FPUJS4A07:0000000F [WRN] The LINQ expression '"where ((([c].FechaDeVenOferta - DateTime.Now).Days > __tiempoVenOferta_OfertasProxVencDesde_1) AndAlso (([c].FechaDeVenOferta - DateTime.Now).Days <= __tiempoVenOferta_OfertasProxVencHasta_2))"' could not be translated and will be evaluated locally. (011792c6)
2020-06-13T15:37:03.6019933-04:00 0HM0FPUJS4A07:0000000F [WRN] The LINQ expression '"Count()"' could not be translated and will be evaluated locally. (011792c6)
2020-06-13T15:37:03.6104327-04:00 0HM0FPUJS4A07:0000000F [WRN] The LINQ expression '"where (([c].FechaDeVenOferta - DateTime.Now).Days > __tiempoVenOferta_OfertaTiempo_1)"' could not be translated and will be evaluated locally. (011792c6)
2020-06-13T15:37:03.6107723-04:00 0HM0FPUJS4A07:0000000F [WRN] The LINQ expression '"Count()"' could not be translated and will be evaluated locally. (011792c6)
2020-06-13T15:40:46.6391362-04:00  [INF] Tarea ejecutada: 13/06/2020 15:40:46 (7ddd3be0)
2020-06-13T16:05:49.9162622-04:00  [INF] Tarea ejecutada: 13/06/2020 16:05:49 (bb16a6ff)
2020-06-13T16:05:51.2587639-04:00  [WRN] No type was specified for the decimal column '"Signo"' on entity type '"AsientosView"'. This will cause values to be silently truncated if they do not fit in the default precision and scale. Explicitly specify the SQL server column type that can accommodate all the values using 'HasColumnType()'. (3a0db05f)
2020-06-13T16:05:51.2589403-04:00  [WRN] No type was specified for the decimal column '"n_importe"' on entity type '"AsientosView"'. This will cause values to be silently truncated if they do not fit in the default precision and scale. Explicitly specify the SQL server column type that can accommodate all the values using 'HasColumnType()'. (3a0db05f)
2020-06-13T16:05:51.2589686-04:00  [WRN] No type was specified for the decimal column '"n_importe_anx"' on entity type '"AsientosView"'. This will cause values to be silently truncated if they do not fit in the default precision and scale. Explicitly specify the SQL server column type that can accommodate all the values using 'HasColumnType()'. (3a0db05f)
2020-06-13T16:05:51.2589859-04:00  [WRN] No type was specified for the decimal column '"n_importemo"' on entity type '"AsientosView"'. This will cause values to be silently truncated if they do not fit in the default precision and scale. Explicitly specify the SQL server column type that can accommodate all the values using 'HasColumnType()'. (3a0db05f)
2020-06-13T16:05:51.5608205-04:00  [INF] Tarea ejecutada: 13/06/2020 16:05:51 (7edfe0c6)
2020-06-13T16:05:51.6032380-04:00  [INF] Tarea ejecutada: 13/06/2020 16:05:51 (7edfe0c6)
2020-06-13T16:05:51.6747307-04:00  [WRN] The LINQ expression '"GroupBy([c.IdcomprobanteNavigation], _Include(queryContext, [c], new [] {[c.IdcomprobanteNavigation], [c.IdcomprobanteNavigation.ConComprobanteoperacion], [c.IdcomprobanteNavigation.ConComprobanteoperacion.IdusuarioNavigation], [c.IdcomprobanteNavigation.ConComprobanteoperacion.IdperiodoNavigation], [c.IdcuentaNavigation], [c.IdcuentaNavigation.IdaperturaNavigation], [c.IdcuentaNavigation.IdaperturaNavigation.IdmascaraNavigation]}, (queryContext, entity, included) => { ... }))"' could not be translated and will be evaluated locally. (011792c6)
2020-06-13T16:05:52.8338118-04:00  [FTL] An error occurred starting the application (dd5d7f26)
System.AggregateException: One or more errors occurred. (One or more errors occurred. (Se ha intentado una operación de socket en un host no accesible)) ---> System.AggregateException: One or more errors occurred. (Se ha intentado una operación de socket en un host no accesible) ---> System.Net.Http.HttpRequestException: Se ha intentado una operación de socket en un host no accesible ---> System.Net.Sockets.SocketException: Se ha intentado una operación de socket en un host no accesible
   at System.Net.Http.ConnectHelper.ConnectAsync(String host, Int32 port, CancellationToken cancellationToken)
   --- End of inner exception stack trace ---
   at System.Net.Http.ConnectHelper.ConnectAsync(String host, Int32 port, CancellationToken cancellationToken)
   at System.Threading.Tasks.ValueTask`1.get_Result()
   at System.Net.Http.HttpConnectionPool.CreateConnectionAsync(HttpRequestMessage request, CancellationToken cancellationToken)
   at System.Threading.Tasks.ValueTask`1.get_Result()
   at System.Net.Http.HttpConnectionPool.WaitForCreatedConnectionAsync(ValueTask`1 creationTask)
   at System.Threading.Tasks.ValueTask`1.get_Result()
   at System.Net.Http.HttpConnectionPool.SendWithRetryAsync(HttpRequestMessage request, Boolean doRequestAuth, CancellationToken cancellationToken)
   at System.Net.Http.HttpConnectionPool.EstablishProxyTunnel(CancellationToken cancellationToken)
   at System.Threading.Tasks.ValueTask`1.get_Result()
   at System.Net.Http.HttpConnectionPool.CreateConnectionAsync(HttpRequestMessage request, CancellationToken cancellationToken)
   at System.Threading.Tasks.ValueTask`1.get_Result()
   at System.Net.Http.HttpConnectionPool.WaitForCreatedConnectionAsync(ValueTask`1 creationTask)
   at System.Threading.Tasks.ValueTask`1.get_Result()
   at System.Net.Http.HttpConnectionPool.SendWithRetryAsync(HttpRequestMessage request, Boolean doRequestAuth, CancellationToken cancellationToken)
   at System.Net.Http.RedirectHandler.SendAsync(HttpRequestMessage request, CancellationToken cancellationToken)
   at System.Net.Http.HttpClient.FinishSendAsyncBuffered(Task`1 sendTask, HttpRequestMessage request, CancellationTokenSource cts, Boolean disposeCts)
   --- End of inner exception stack trace ---
   at System.Threading.Tasks.Task`1.GetResultCore(Boolean waitCompletionNotification)
   at ImportadorDatos.Jobs.ImportarVersat.ImportarCuentasAsync() in D:\Alejandro\New Folder\Git\opplat\src\ImportadorDatos\Jobs\ImportarVersat.cs:line 66
   at ImportadorDatos.HostedServices.UpdateCuentasHostedService.ExecuteAsync(CancellationToken stoppingToken) in D:\Alejandro\New Folder\Git\opplat\src\ImportadorDatos\HostedServices\UpdateCuentasHostedService.cs:line 33
   at Microsoft.AspNetCore.Hosting.Internal.HostedServiceExecutor.ExecuteAsync(Func`2 callback)
   --- End of inner exception stack trace ---
   at Microsoft.AspNetCore.Hosting.Internal.HostedServiceExecutor.ExecuteAsync(Func`2 callback)
   at Microsoft.AspNetCore.Hosting.Internal.HostedServiceExecutor.StartAsync(CancellationToken token)
---> (Inner Exception #0) System.AggregateException: One or more errors occurred. (Se ha intentado una operación de socket en un host no accesible) ---> System.Net.Http.HttpRequestException: Se ha intentado una operación de socket en un host no accesible ---> System.Net.Sockets.SocketException: Se ha intentado una operación de socket en un host no accesible
   at System.Net.Http.ConnectHelper.ConnectAsync(String host, Int32 port, CancellationToken cancellationToken)
   --- End of inner exception stack trace ---
   at System.Net.Http.ConnectHelper.ConnectAsync(String host, Int32 port, CancellationToken cancellationToken)
   at System.Threading.Tasks.ValueTask`1.get_Result()
   at System.Net.Http.HttpConnectionPool.CreateConnectionAsync(HttpRequestMessage request, CancellationToken cancellationToken)
   at System.Threading.Tasks.ValueTask`1.get_Result()
   at System.Net.Http.HttpConnectionPool.WaitForCreatedConnectionAsync(ValueTask`1 creationTask)
   at System.Threading.Tasks.ValueTask`1.get_Result()
   at System.Net.Http.HttpConnectionPool.SendWithRetryAsync(HttpRequestMessage request, Boolean doRequestAuth, CancellationToken cancellationToken)
   at System.Net.Http.HttpConnectionPool.EstablishProxyTunnel(CancellationToken cancellationToken)
   at System.Threading.Tasks.ValueTask`1.get_Result()
   at System.Net.Http.HttpConnectionPool.CreateConnectionAsync(HttpRequestMessage request, CancellationToken cancellationToken)
   at System.Threading.Tasks.ValueTask`1.get_Result()
   at System.Net.Http.HttpConnectionPool.WaitForCreatedConnectionAsync(ValueTask`1 creationTask)
   at System.Threading.Tasks.ValueTask`1.get_Result()
   at System.Net.Http.HttpConnectionPool.SendWithRetryAsync(HttpRequestMessage request, Boolean doRequestAuth, CancellationToken cancellationToken)
   at System.Net.Http.RedirectHandler.SendAsync(HttpRequestMessage request, CancellationToken cancellationToken)
   at System.Net.Http.HttpClient.FinishSendAsyncBuffered(Task`1 sendTask, HttpRequestMessage request, CancellationTokenSource cts, Boolean disposeCts)
   --- End of inner exception stack trace ---
   at System.Threading.Tasks.Task`1.GetResultCore(Boolean waitCompletionNotification)
   at ImportadorDatos.Jobs.ImportarVersat.ImportarCuentasAsync() in D:\Alejandro\New Folder\Git\opplat\src\ImportadorDatos\Jobs\ImportarVersat.cs:line 66
   at ImportadorDatos.HostedServices.UpdateCuentasHostedService.ExecuteAsync(CancellationToken stoppingToken) in D:\Alejandro\New Folder\Git\opplat\src\ImportadorDatos\HostedServices\UpdateCuentasHostedService.cs:line 33
   at Microsoft.AspNetCore.Hosting.Internal.HostedServiceExecutor.ExecuteAsync(Func`2 callback)
---> (Inner Exception #0) System.Net.Http.HttpRequestException: Se ha intentado una operación de socket en un host no accesible ---> System.Net.Sockets.SocketException: Se ha intentado una operación de socket en un host no accesible
   at System.Net.Http.ConnectHelper.ConnectAsync(String host, Int32 port, CancellationToken cancellationToken)
   --- End of inner exception stack trace ---
   at System.Net.Http.ConnectHelper.ConnectAsync(String host, Int32 port, CancellationToken cancellationToken)
   at System.Threading.Tasks.ValueTask`1.get_Result()
   at System.Net.Http.HttpConnectionPool.CreateConnectionAsync(HttpRequestMessage request, CancellationToken cancellationToken)
   at System.Threading.Tasks.ValueTask`1.get_Result()
   at System.Net.Http.HttpConnectionPool.WaitForCreatedConnectionAsync(ValueTask`1 creationTask)
   at System.Threading.Tasks.ValueTask`1.get_Result()
   at System.Net.Http.HttpConnectionPool.SendWithRetryAsync(HttpRequestMessage request, Boolean doRequestAuth, CancellationToken cancellationToken)
   at System.Net.Http.HttpConnectionPool.EstablishProxyTunnel(CancellationToken cancellationToken)
   at System.Threading.Tasks.ValueTask`1.get_Result()
   at System.Net.Http.HttpConnectionPool.CreateConnectionAsync(HttpRequestMessage request, CancellationToken cancellationToken)
   at System.Threading.Tasks.ValueTask`1.get_Result()
   at System.Net.Http.HttpConnectionPool.WaitForCreatedConnectionAsync(ValueTask`1 creationTask)
   at System.Threading.Tasks.ValueTask`1.get_Result()
   at System.Net.Http.HttpConnectionPool.SendWithRetryAsync(HttpRequestMessage request, Boolean doRequestAuth, CancellationToken cancellationToken)
   at System.Net.Http.RedirectHandler.SendAsync(HttpRequestMessage request, CancellationToken cancellationToken)
   at System.Net.Http.HttpClient.FinishSendAsyncBuffered(Task`1 sendTask, HttpRequestMessage request, CancellationTokenSource cts, Boolean disposeCts)<---
<---

2020-06-13T16:06:31.9484514-04:00 0HM0FQL116R8N:00000003 [WRN] The LINQ expression '"where (([c].FechaDeVenOferta - DateTime.Now).Days < __tiempoVenOferta_OfertasVencidas_1)"' could not be translated and will be evaluated locally. (011792c6)
2020-06-13T16:06:31.9503621-04:00 0HM0FQL116R8N:00000003 [WRN] The LINQ expression '"Count()"' could not be translated and will be evaluated locally. (011792c6)
2020-06-13T16:06:31.9555434-04:00 0HM0FQL116R8O:00000002 [WRN] The LINQ expression '"where (([c].FechaDeVenOferta - DateTime.Now).Days < __tiempoVenOferta_OfertasVencidas_1)"' could not be translated and will be evaluated locally. (011792c6)
2020-06-13T16:06:31.9559054-04:00 0HM0FQL116R8O:00000002 [WRN] The LINQ expression '"Count()"' could not be translated and will be evaluated locally. (011792c6)
2020-06-13T16:06:31.9979775-04:00 0HM0FQL116R8J:00000006 [WRN] The LINQ expression '"where ((([c].FechaDeVenOferta - DateTime.Now).Days >= __tiempoVenOferta_OfertasCasiVencDesde_1) AndAlso (([c].FechaDeVenOferta - DateTime.Now).Days <= __tiempoVenOferta_OfertasCasiVencHasta_2))"' could not be translated and will be evaluated locally. (011792c6)
2020-06-13T16:06:31.9983784-04:00 0HM0FQL116R8J:00000006 [WRN] The LINQ expression '"Count()"' could not be translated and will be evaluated locally. (011792c6)
2020-06-13T16:06:32.0035533-04:00 0HM0FQL116R8O:00000002 [WRN] The LINQ expression '"where ((([c].FechaDeVenOferta - DateTime.Now).Days >= __tiempoVenOferta_OfertasCasiVencDesde_1) AndAlso (([c].FechaDeVenOferta - DateTime.Now).Days <= __tiempoVenOferta_OfertasCasiVencHasta_2))"' could not be translated and will be evaluated locally. (011792c6)
2020-06-13T16:06:32.0040846-04:00 0HM0FQL116R8O:00000002 [WRN] The LINQ expression '"Count()"' could not be translated and will be evaluated locally. (011792c6)
2020-06-13T16:06:32.0080963-04:00 0HM0FQL116R8K:00000004 [WRN] The LINQ expression '"where ((([c].FechaDeVenOferta - DateTime.Now).Days > __tiempoVenOferta_OfertasProxVencDesde_1) AndAlso (([c].FechaDeVenOferta - DateTime.Now).Days <= __tiempoVenOferta_OfertasProxVencHasta_2))"' could not be translated and will be evaluated locally. (011792c6)
2020-06-13T16:06:32.0085550-04:00 0HM0FQL116R8K:00000004 [WRN] The LINQ expression '"Count()"' could not be translated and will be evaluated locally. (011792c6)
2020-06-13T16:06:32.0162460-04:00 0HM0FQL116R8N:00000003 [WRN] The LINQ expression '"where (([c].FechaDeVenOferta - DateTime.Now).Days > __tiempoVenOferta_OfertaTiempo_1)"' could not be translated and will be evaluated locally. (011792c6)
2020-06-13T16:06:32.0168361-04:00 0HM0FQL116R8N:00000003 [WRN] The LINQ expression '"Count()"' could not be translated and will be evaluated locally. (011792c6)
2020-06-13T16:06:32.0235553-04:00 0HM0FQL116R8J:00000006 [WRN] The LINQ expression '"where (([c].FechaDeVenOferta - DateTime.Now).Days > __tiempoVenOferta_OfertaTiempo_1)"' could not be translated and will be evaluated locally. (011792c6)
2020-06-13T16:06:32.0242002-04:00 0HM0FQL116R8J:00000006 [WRN] The LINQ expression '"Count()"' could not be translated and will be evaluated locally. (011792c6)
2020-06-13T16:08:59.6279003-04:00 0HM0FQL116R8Q:00000002 [WRN] The Include operation for navigation '"[p].Detalles.Concepto"' is unnecessary and was ignored because the navigation is not reachable in the final query results. See https://go.microsoft.com/fwlink/?linkid=850303 for more information. (10efaee6)
2020-06-13T16:09:04.1474359-04:00 0HM0FQL116R8P:00000004 [WRN] The LINQ expression '"where (([c].FechaDeVenOferta - DateTime.Now).Days < __8__locals1_tiempoVenOferta_OfertasVencidas_2)"' could not be translated and will be evaluated locally. (011792c6)
2020-06-13T16:09:04.1476490-04:00 0HM0FQL116R8P:00000004 [WRN] The LINQ expression '"Count()"' could not be translated and will be evaluated locally. (011792c6)
2020-06-13T16:09:04.1753329-04:00 0HM0FQL116R8P:00000004 [WRN] The LINQ expression '"where (([c].FechaDeVenOferta - DateTime.Now).Days < __tiempoVenOferta_OfertasVencidas_1)"' could not be translated and will be evaluated locally. (011792c6)
2020-06-13T16:09:04.1756109-04:00 0HM0FQL116R8P:00000004 [WRN] The LINQ expression '"Count()"' could not be translated and will be evaluated locally. (011792c6)
2020-06-13T16:09:04.1811695-04:00 0HM0FQL116R8P:00000004 [WRN] The LINQ expression '"where (([c].FechaDeVenOferta - DateTime.Now).Days < __tiempoVenOferta_OfertasVencidas_1)"' could not be translated and will be evaluated locally. (011792c6)
2020-06-13T16:09:04.1813519-04:00 0HM0FQL116R8P:00000004 [WRN] The LINQ expression '"Count()"' could not be translated and will be evaluated locally. (011792c6)
2020-06-13T16:09:51.6683179-04:00 0HM0FQL116R8V:00000001 [WRN] The Include operation for navigation '"[e].CuentasBancarias"' is unnecessary and was ignored because the navigation is not reachable in the final query results. See https://go.microsoft.com/fwlink/?linkid=850303 for more information. (10efaee6)
2020-06-13T16:09:51.6686126-04:00 0HM0FQL116R8V:00000001 [WRN] The Include operation for navigation '"[e].Telefonos"' is unnecessary and was ignored because the navigation is not reachable in the final query results. See https://go.microsoft.com/fwlink/?linkid=850303 for more information. (10efaee6)
2020-06-13T16:09:51.7139537-04:00 0HM0FQL116R8Q:0000000C [WRN] The LINQ expression '"where ([t].Id == [c].AdminContratoId)"' could not be translated and will be evaluated locally. (011792c6)
2020-06-13T16:09:51.7146540-04:00 0HM0FQL116R8Q:0000000C [WRN] The LINQ expression '"FirstOrDefault()"' could not be translated and will be evaluated locally. (011792c6)
2020-06-13T16:09:51.7151725-04:00 0HM0FQL116R8Q:0000000C [WRN] The LINQ expression '"where ([s].ContratoId == [c].Id)"' could not be translated and will be evaluated locally. (011792c6)
2020-06-13T16:09:51.7158562-04:00 0HM0FQL116R8Q:0000000C [WRN] The LINQ expression '"where ([d].Id == [s].DictaminadorContratoId)"' could not be translated and will be evaluated locally. (011792c6)
2020-06-13T16:09:51.7161370-04:00 0HM0FQL116R8Q:0000000C [WRN] The LINQ expression '"FirstOrDefault()"' could not be translated and will be evaluated locally. (011792c6)
2020-06-13T16:10:52.8185974-04:00  [INF] Tarea ejecutada: 13/06/2020 16:10:52 (f3cf2631)
2020-06-13T16:15:54.0300780-04:00  [INF] Tarea ejecutada: 13/06/2020 16:15:54 (ed221e9c)
2020-06-13T16:20:55.1099130-04:00  [INF] Tarea ejecutada: 13/06/2020 16:20:55 (bbc55717)
2020-06-13T16:25:56.3897487-04:00  [INF] Tarea ejecutada: 13/06/2020 16:25:56 (c459d766)
2020-06-13T16:26:57.4842860-04:00 0HM0FQL116R96:00000002 [WRN] The LINQ expression '"where ([t].Id == [c].AdminContratoId)"' could not be translated and will be evaluated locally. (011792c6)
2020-06-13T16:26:57.4844371-04:00 0HM0FQL116R96:00000002 [WRN] The LINQ expression '"FirstOrDefault()"' could not be translated and will be evaluated locally. (011792c6)
2020-06-13T16:26:57.4845272-04:00 0HM0FQL116R96:00000002 [WRN] The LINQ expression '"where ([s].ContratoId == [c].Id)"' could not be translated and will be evaluated locally. (011792c6)
2020-06-13T16:26:57.4846866-04:00 0HM0FQL116R96:00000002 [WRN] The LINQ expression '"where ([d].Id == [s].DictaminadorContratoId)"' could not be translated and will be evaluated locally. (011792c6)
2020-06-13T16:26:57.4847696-04:00 0HM0FQL116R96:00000002 [WRN] The LINQ expression '"FirstOrDefault()"' could not be translated and will be evaluated locally. (011792c6)
2020-06-13T16:28:10.4380588-04:00 0HM0FQL116R97:00000005 [WRN] The LINQ expression '"where (([c].FechaDeVenOferta - DateTime.Now).Days < __tiempoVenOferta_OfertasVencidas_1)"' could not be translated and will be evaluated locally. (011792c6)
2020-06-13T16:28:10.4386359-04:00 0HM0FQL116R97:00000005 [WRN] The LINQ expression '"Count()"' could not be translated and will be evaluated locally. (011792c6)
2020-06-13T16:28:10.4482162-04:00 0HM0FQL116R97:00000005 [WRN] The LINQ expression '"where ((([c].FechaDeVenOferta - DateTime.Now).Days >= __tiempoVenOferta_OfertasCasiVencDesde_1) AndAlso (([c].FechaDeVenOferta - DateTime.Now).Days <= __tiempoVenOferta_OfertasCasiVencHasta_2))"' could not be translated and will be evaluated locally. (011792c6)
2020-06-13T16:28:10.4486842-04:00 0HM0FQL116R97:00000005 [WRN] The LINQ expression '"Count()"' could not be translated and will be evaluated locally. (011792c6)
2020-06-13T16:28:10.4578607-04:00 0HM0FQL116R97:00000005 [WRN] The LINQ expression '"where ((([c].FechaDeVenOferta - DateTime.Now).Days > __tiempoVenOferta_OfertasProxVencDesde_1) AndAlso (([c].FechaDeVenOferta - DateTime.Now).Days <= __tiempoVenOferta_OfertasProxVencHasta_2))"' could not be translated and will be evaluated locally. (011792c6)
2020-06-13T16:28:10.4582782-04:00 0HM0FQL116R97:00000005 [WRN] The LINQ expression '"Count()"' could not be translated and will be evaluated locally. (011792c6)
2020-06-13T16:28:10.4666833-04:00 0HM0FQL116R97:00000005 [WRN] The LINQ expression '"where (([c].FechaDeVenOferta - DateTime.Now).Days > __tiempoVenOferta_OfertaTiempo_1)"' could not be translated and will be evaluated locally. (011792c6)
2020-06-13T16:28:10.4670364-04:00 0HM0FQL116R97:00000005 [WRN] The LINQ expression '"Count()"' could not be translated and will be evaluated locally. (011792c6)
2020-06-13T16:28:10.4829504-04:00 0HM0FQL116R99:00000004 [WRN] The LINQ expression '"where ([t].Id == [c].AdminContratoId)"' could not be translated and will be evaluated locally. (011792c6)
2020-06-13T16:28:10.4832775-04:00 0HM0FQL116R99:00000004 [WRN] The LINQ expression '"FirstOrDefault()"' could not be translated and will be evaluated locally. (011792c6)
2020-06-13T16:28:10.4834993-04:00 0HM0FQL116R99:00000004 [WRN] The LINQ expression '"where ([s].ContratoId == [c].Id)"' could not be translated and will be evaluated locally. (011792c6)
2020-06-13T16:28:10.4839050-04:00 0HM0FQL116R99:00000004 [WRN] The LINQ expression '"where ([d].Id == [s].DictaminadorContratoId)"' could not be translated and will be evaluated locally. (011792c6)
2020-06-13T16:28:10.4841109-04:00 0HM0FQL116R99:00000004 [WRN] The LINQ expression '"FirstOrDefault()"' could not be translated and will be evaluated locally. (011792c6)
2020-06-13T16:28:23.4006462-04:00 0HM0FQL116R9A:00000012 [WRN] The LINQ expression '"where ([t].Id == [c].AdminContratoId)"' could not be translated and will be evaluated locally. (011792c6)
2020-06-13T16:28:23.4008766-04:00 0HM0FQL116R9A:00000012 [WRN] The LINQ expression '"FirstOrDefault()"' could not be translated and will be evaluated locally. (011792c6)
2020-06-13T16:28:23.4010332-04:00 0HM0FQL116R9A:00000012 [WRN] The LINQ expression '"where ([s].ContratoId == [c].Id)"' could not be translated and will be evaluated locally. (011792c6)
2020-06-13T16:28:23.4013161-04:00 0HM0FQL116R9A:00000012 [WRN] The LINQ expression '"where ([d].Id == [s].DictaminadorContratoId)"' could not be translated and will be evaluated locally. (011792c6)
2020-06-13T16:28:23.4014669-04:00 0HM0FQL116R9A:00000012 [WRN] The LINQ expression '"FirstOrDefault()"' could not be translated and will be evaluated locally. (011792c6)
2020-06-13T16:28:23.5192614-04:00 0HM0FQL116R9A:00000014 [WRN] The LINQ expression '"where (([c].FechaVenContrato - DateTime.Now).Days < __tiempoVenContrato_ContratosVencidos_1)"' could not be translated and will be evaluated locally. (011792c6)
2020-06-13T16:28:23.5194629-04:00 0HM0FQL116R9A:00000014 [WRN] The LINQ expression '"Count()"' could not be translated and will be evaluated locally. (011792c6)
2020-06-13T16:28:23.5258239-04:00 0HM0FQL116R9A:00000014 [WRN] The LINQ expression '"where ((([c].FechaVenContrato - DateTime.Now).Days >= __tiempoVenContrato_ContratosCasiVencDesde_1) AndAlso (([c].FechaVenContrato - DateTime.Now).Days <= __tiempoVenContrato_ContratosCasiVencHasta_2))"' could not be translated and will be evaluated locally. (011792c6)
2020-06-13T16:28:23.5263165-04:00 0HM0FQL116R9A:00000014 [WRN] The LINQ expression '"Count()"' could not be translated and will be evaluated locally. (011792c6)
2020-06-13T16:28:23.5318053-04:00 0HM0FQL116R9A:00000014 [WRN] The LINQ expression '"where ((([c].FechaVenContrato - DateTime.Now).Days > __tiempoVenContrato_ContratosProxVencerDesde_1) AndAlso (([c].FechaVenContrato - DateTime.Now).Days <= __tiempoVenContrato_ContratosProxVencerHasta_2))"' could not be translated and will be evaluated locally. (011792c6)
2020-06-13T16:28:23.5320473-04:00 0HM0FQL116R9A:00000014 [WRN] The LINQ expression '"Count()"' could not be translated and will be evaluated locally. (011792c6)
2020-06-13T16:28:23.5368717-04:00 0HM0FQL116R9A:00000014 [WRN] The LINQ expression '"where (([c].FechaVenContrato - DateTime.Now).Days > __tiempoVenContrato_ContratoTiempo_1)"' could not be translated and will be evaluated locally. (011792c6)
2020-06-13T16:28:23.5370697-04:00 0HM0FQL116R9A:00000014 [WRN] The LINQ expression '"Count()"' could not be translated and will be evaluated locally. (011792c6)
2020-06-13T16:30:57.6135493-04:00  [INF] Tarea ejecutada: 13/06/2020 16:30:57 (620ccb5f)
2020-06-13T16:36:06.5899351-04:00  [INF] Tarea ejecutada: 13/06/2020 16:36:06 (9e7a2200)
2020-06-13T16:41:08.0714559-04:00  [INF] Tarea ejecutada: 13/06/2020 16:41:08 (f7c2659e)
2020-06-13T16:46:09.2771364-04:00  [INF] Tarea ejecutada: 13/06/2020 16:46:09 (6a24308e)
2020-06-13T16:51:10.4319320-04:00  [INF] Tarea ejecutada: 13/06/2020 16:51:10 (b6e43f51)
2020-06-13T16:56:11.6914786-04:00  [INF] Tarea ejecutada: 13/06/2020 16:56:11 (6a09cace)
2020-06-13T17:25:34.4978393-04:00  [INF] Tarea ejecutada: 13/06/2020 17:25:34 (127a1065)
2020-06-13T17:25:35.9089762-04:00  [WRN] No type was specified for the decimal column '"Signo"' on entity type '"AsientosView"'. This will cause values to be silently truncated if they do not fit in the default precision and scale. Explicitly specify the SQL server column type that can accommodate all the values using 'HasColumnType()'. (3a0db05f)
2020-06-13T17:25:35.9092532-04:00  [WRN] No type was specified for the decimal column '"n_importe"' on entity type '"AsientosView"'. This will cause values to be silently truncated if they do not fit in the default precision and scale. Explicitly specify the SQL server column type that can accommodate all the values using 'HasColumnType()'. (3a0db05f)
2020-06-13T17:25:35.9093025-04:00  [WRN] No type was specified for the decimal column '"n_importe_anx"' on entity type '"AsientosView"'. This will cause values to be silently truncated if they do not fit in the default precision and scale. Explicitly specify the SQL server column type that can accommodate all the values using 'HasColumnType()'. (3a0db05f)
2020-06-13T17:25:35.9093602-04:00  [WRN] No type was specified for the decimal column '"n_importemo"' on entity type '"AsientosView"'. This will cause values to be silently truncated if they do not fit in the default precision and scale. Explicitly specify the SQL server column type that can accommodate all the values using 'HasColumnType()'. (3a0db05f)
2020-06-13T17:25:38.1578157-04:00  [INF] Tarea ejecutada: 13/06/2020 17:25:38 (d1b40eda)
2020-06-13T17:25:38.3009350-04:00  [INF] Tarea ejecutada: 13/06/2020 17:25:38 (d1b40eda)
2020-06-13T17:25:38.3810494-04:00  [WRN] The LINQ expression '"GroupBy([c.IdcomprobanteNavigation], _Include(queryContext, [c], new [] {[c.IdcomprobanteNavigation], [c.IdcomprobanteNavigation.ConComprobanteoperacion], [c.IdcomprobanteNavigation.ConComprobanteoperacion.IdusuarioNavigation], [c.IdcomprobanteNavigation.ConComprobanteoperacion.IdperiodoNavigation], [c.IdcuentaNavigation], [c.IdcuentaNavigation.IdaperturaNavigation], [c.IdcuentaNavigation.IdaperturaNavigation.IdmascaraNavigation]}, (queryContext, entity, included) => { ... }))"' could not be translated and will be evaluated locally. (011792c6)
2020-06-13T17:25:40.1074671-04:00  [FTL] An error occurred starting the application (dd5d7f26)
System.AggregateException: One or more errors occurred. (One or more errors occurred. (Se ha intentado una operación de socket en un host no accesible)) ---> System.AggregateException: One or more errors occurred. (Se ha intentado una operación de socket en un host no accesible) ---> System.Net.Http.HttpRequestException: Se ha intentado una operación de socket en un host no accesible ---> System.Net.Sockets.SocketException: Se ha intentado una operación de socket en un host no accesible
   at System.Net.Http.ConnectHelper.ConnectAsync(String host, Int32 port, CancellationToken cancellationToken)
   --- End of inner exception stack trace ---
   at System.Net.Http.ConnectHelper.ConnectAsync(String host, Int32 port, CancellationToken cancellationToken)
   at System.Threading.Tasks.ValueTask`1.get_Result()
   at System.Net.Http.HttpConnectionPool.CreateConnectionAsync(HttpRequestMessage request, CancellationToken cancellationToken)
   at System.Threading.Tasks.ValueTask`1.get_Result()
   at System.Net.Http.HttpConnectionPool.WaitForCreatedConnectionAsync(ValueTask`1 creationTask)
   at System.Threading.Tasks.ValueTask`1.get_Result()
   at System.Net.Http.HttpConnectionPool.SendWithRetryAsync(HttpRequestMessage request, Boolean doRequestAuth, CancellationToken cancellationToken)
   at System.Net.Http.HttpConnectionPool.EstablishProxyTunnel(CancellationToken cancellationToken)
   at System.Threading.Tasks.ValueTask`1.get_Result()
   at System.Net.Http.HttpConnectionPool.CreateConnectionAsync(HttpRequestMessage request, CancellationToken cancellationToken)
   at System.Threading.Tasks.ValueTask`1.get_Result()
   at System.Net.Http.HttpConnectionPool.WaitForCreatedConnectionAsync(ValueTask`1 creationTask)
   at System.Threading.Tasks.ValueTask`1.get_Result()
   at System.Net.Http.HttpConnectionPool.SendWithRetryAsync(HttpRequestMessage request, Boolean doRequestAuth, CancellationToken cancellationToken)
   at System.Net.Http.RedirectHandler.SendAsync(HttpRequestMessage request, CancellationToken cancellationToken)
   at System.Net.Http.HttpClient.FinishSendAsyncBuffered(Task`1 sendTask, HttpRequestMessage request, CancellationTokenSource cts, Boolean disposeCts)
   --- End of inner exception stack trace ---
   at System.Threading.Tasks.Task`1.GetResultCore(Boolean waitCompletionNotification)
   at ImportadorDatos.Jobs.ImportarVersat.ImportarCuentasAsync() in D:\Alejandro\New Folder\Git\opplat\src\ImportadorDatos\Jobs\ImportarVersat.cs:line 66
   at ImportadorDatos.HostedServices.UpdateCuentasHostedService.ExecuteAsync(CancellationToken stoppingToken) in D:\Alejandro\New Folder\Git\opplat\src\ImportadorDatos\HostedServices\UpdateCuentasHostedService.cs:line 33
   at Microsoft.AspNetCore.Hosting.Internal.HostedServiceExecutor.ExecuteAsync(Func`2 callback)
   --- End of inner exception stack trace ---
   at Microsoft.AspNetCore.Hosting.Internal.HostedServiceExecutor.ExecuteAsync(Func`2 callback)
   at Microsoft.AspNetCore.Hosting.Internal.HostedServiceExecutor.StartAsync(CancellationToken token)
---> (Inner Exception #0) System.AggregateException: One or more errors occurred. (Se ha intentado una operación de socket en un host no accesible) ---> System.Net.Http.HttpRequestException: Se ha intentado una operación de socket en un host no accesible ---> System.Net.Sockets.SocketException: Se ha intentado una operación de socket en un host no accesible
   at System.Net.Http.ConnectHelper.ConnectAsync(String host, Int32 port, CancellationToken cancellationToken)
   --- End of inner exception stack trace ---
   at System.Net.Http.ConnectHelper.ConnectAsync(String host, Int32 port, CancellationToken cancellationToken)
   at System.Threading.Tasks.ValueTask`1.get_Result()
   at System.Net.Http.HttpConnectionPool.CreateConnectionAsync(HttpRequestMessage request, CancellationToken cancellationToken)
   at System.Threading.Tasks.ValueTask`1.get_Result()
   at System.Net.Http.HttpConnectionPool.WaitForCreatedConnectionAsync(ValueTask`1 creationTask)
   at System.Threading.Tasks.ValueTask`1.get_Result()
   at System.Net.Http.HttpConnectionPool.SendWithRetryAsync(HttpRequestMessage request, Boolean doRequestAuth, CancellationToken cancellationToken)
   at System.Net.Http.HttpConnectionPool.EstablishProxyTunnel(CancellationToken cancellationToken)
   at System.Threading.Tasks.ValueTask`1.get_Result()
   at System.Net.Http.HttpConnectionPool.CreateConnectionAsync(HttpRequestMessage request, CancellationToken cancellationToken)
   at System.Threading.Tasks.ValueTask`1.get_Result()
   at System.Net.Http.HttpConnectionPool.WaitForCreatedConnectionAsync(ValueTask`1 creationTask)
   at System.Threading.Tasks.ValueTask`1.get_Result()
   at System.Net.Http.HttpConnectionPool.SendWithRetryAsync(HttpRequestMessage request, Boolean doRequestAuth, CancellationToken cancellationToken)
   at System.Net.Http.RedirectHandler.SendAsync(HttpRequestMessage request, CancellationToken cancellationToken)
   at System.Net.Http.HttpClient.FinishSendAsyncBuffered(Task`1 sendTask, HttpRequestMessage request, CancellationTokenSource cts, Boolean disposeCts)
   --- End of inner exception stack trace ---
   at System.Threading.Tasks.Task`1.GetResultCore(Boolean waitCompletionNotification)
   at ImportadorDatos.Jobs.ImportarVersat.ImportarCuentasAsync() in D:\Alejandro\New Folder\Git\opplat\src\ImportadorDatos\Jobs\ImportarVersat.cs:line 66
   at ImportadorDatos.HostedServices.UpdateCuentasHostedService.ExecuteAsync(CancellationToken stoppingToken) in D:\Alejandro\New Folder\Git\opplat\src\ImportadorDatos\HostedServices\UpdateCuentasHostedService.cs:line 33
   at Microsoft.AspNetCore.Hosting.Internal.HostedServiceExecutor.ExecuteAsync(Func`2 callback)
---> (Inner Exception #0) System.Net.Http.HttpRequestException: Se ha intentado una operación de socket en un host no accesible ---> System.Net.Sockets.SocketException: Se ha intentado una operación de socket en un host no accesible
   at System.Net.Http.ConnectHelper.ConnectAsync(String host, Int32 port, CancellationToken cancellationToken)
   --- End of inner exception stack trace ---
   at System.Net.Http.ConnectHelper.ConnectAsync(String host, Int32 port, CancellationToken cancellationToken)
   at System.Threading.Tasks.ValueTask`1.get_Result()
   at System.Net.Http.HttpConnectionPool.CreateConnectionAsync(HttpRequestMessage request, CancellationToken cancellationToken)
   at System.Threading.Tasks.ValueTask`1.get_Result()
   at System.Net.Http.HttpConnectionPool.WaitForCreatedConnectionAsync(ValueTask`1 creationTask)
   at System.Threading.Tasks.ValueTask`1.get_Result()
   at System.Net.Http.HttpConnectionPool.SendWithRetryAsync(HttpRequestMessage request, Boolean doRequestAuth, CancellationToken cancellationToken)
   at System.Net.Http.HttpConnectionPool.EstablishProxyTunnel(CancellationToken cancellationToken)
   at System.Threading.Tasks.ValueTask`1.get_Result()
   at System.Net.Http.HttpConnectionPool.CreateConnectionAsync(HttpRequestMessage request, CancellationToken cancellationToken)
   at System.Threading.Tasks.ValueTask`1.get_Result()
   at System.Net.Http.HttpConnectionPool.WaitForCreatedConnectionAsync(ValueTask`1 creationTask)
   at System.Threading.Tasks.ValueTask`1.get_Result()
   at System.Net.Http.HttpConnectionPool.SendWithRetryAsync(HttpRequestMessage request, Boolean doRequestAuth, CancellationToken cancellationToken)
   at System.Net.Http.RedirectHandler.SendAsync(HttpRequestMessage request, CancellationToken cancellationToken)
   at System.Net.Http.HttpClient.FinishSendAsyncBuffered(Task`1 sendTask, HttpRequestMessage request, CancellationTokenSource cts, Boolean disposeCts)<---
<---

2020-06-13T17:25:48.8466740-04:00 0HM0FS1AJ1UC3:00000005 [WRN] The LINQ expression '"where (([c].FechaDeVenOferta - DateTime.Now).Days < __8__locals1_tiempoVenOferta_OfertasVencidas_2)"' could not be translated and will be evaluated locally. (011792c6)
2020-06-13T17:25:48.8475919-04:00 0HM0FS1AJ1UC3:00000005 [WRN] The LINQ expression '"Count()"' could not be translated and will be evaluated locally. (011792c6)
2020-06-13T17:25:48.8783073-04:00 0HM0FS1AJ1UC3:00000005 [WRN] The LINQ expression '"where (([c].FechaDeVenOferta - DateTime.Now).Days < __tiempoVenOferta_OfertasVencidas_1)"' could not be translated and will be evaluated locally. (011792c6)
2020-06-13T17:25:48.8785533-04:00 0HM0FS1AJ1UC3:00000005 [WRN] The LINQ expression '"Count()"' could not be translated and will be evaluated locally. (011792c6)
2020-06-13T17:25:48.8843492-04:00 0HM0FS1AJ1UC3:00000005 [WRN] The LINQ expression '"where (([c].FechaDeVenOferta - DateTime.Now).Days < __tiempoVenOferta_OfertasVencidas_1)"' could not be translated and will be evaluated locally. (011792c6)
2020-06-13T17:25:48.8845691-04:00 0HM0FS1AJ1UC3:00000005 [WRN] The LINQ expression '"Count()"' could not be translated and will be evaluated locally. (011792c6)
2020-06-13T17:25:53.9123855-04:00 0HM0FS1AJ1UC5:00000001 [WRN] The Include operation for navigation '"[e].CuentasBancarias"' is unnecessary and was ignored because the navigation is not reachable in the final query results. See https://go.microsoft.com/fwlink/?linkid=850303 for more information. (10efaee6)
2020-06-13T17:25:53.9124570-04:00 0HM0FS1AJ1UC5:00000001 [WRN] The Include operation for navigation '"[e].Telefonos"' is unnecessary and was ignored because the navigation is not reachable in the final query results. See https://go.microsoft.com/fwlink/?linkid=850303 for more information. (10efaee6)
2020-06-13T17:30:40.0788345-04:00  [INF] Tarea ejecutada: 13/06/2020 17:30:40 (a490ce1e)
2020-06-13T19:17:28.2495024-04:00  [INF] Tarea ejecutada: 13/06/2020 19:17:28 (de5765f2)
